docs/docs/guides/build/external-pipelines/azureml-pipeline.md

```diff
@@ -0,0 +1,99 @@
+---
+title: Build pipelines with Azure Machine Learning
+description: "Learn to integrate Dagster Pipes with Azure Machine Learning to launch external code from Dagster assets."
+sidebar_position: 35
+---
+
+import ScaffoldProject from '@site/docs/partials/\_ScaffoldProject.md';
+
+This article covers how to use [Dagster Pipes](/guides/build/external-pipelines) to submit jobs to [Azure Machine Learning](https://azure.microsoft.com/en-us/products/machine-learning).
+
+The [dagster-azure](/api/libraries/dagster-azure) integration library provides the `PipesAzureMLClient` resource, which can be used to launch Azure ML jobs from Dagster assets and ops. Dagster can receive events such as logs, asset checks, or asset materializations from jobs launched with this client. The client requires minimal code changes to your Azure ML jobs.
+
+## Prerequisites
+
+To run the examples, you'll need to:
+
+- Create a new Dagster project:
+  <ScaffoldProject />
+- Install the necessary Python libraries:
+
+<Tabs groupId="package-manager">
+   <TabItem value="uv" label="uv">
+      Install the required dependencies:
+
+         ```shell
+         uv add dagster-azure
+         ```
+
+   </TabItem>
+
+   <TabItem value="pip" label="pip">
+      Install the required dependencies:
+
+         ```shell
+         pip install dagster-azure
+         ```
+
+   </TabItem>
+</Tabs>
+
+- In Azure, you'll need:
+  - An existing Azure ML workspace
+  - An Azure Blob Data Storage Container to be used by dagster. The recommended way to work with dagster-pipes and AzureML is to use Azure Blob Data Storage to communicate between the Dagster orchestrator and the AzureML job.
+
+## Step 1: Create an AzureML environment for dagster pipes.
+
+Your AzureML job will require an AzureML environment that contains the `dagtster-pipes` library. Since the AzureML job will be communicating with the dagster orchestrator via Azure Blob Storage, we will also need to install the `azure-identity` and `azure-storage-blob` python libraries.
```

Comment:
```suggestion
Your Azure ML job will require an Azure ML environment that contains the `dagtster-pipes` library. Since the Azure ML job will be communicating with the Dagster orchestrator via Azure Blob Storage, we will also need to install the `azure-identity` and `azure-storage-blob` Python libraries.
```